spec/predicates/tool-assessment.md

```diff
@@ -0,0 +1,286 @@
+# Predicate type: Tool Assessment
+
+Type URI: https://in-toto.io/attestation/tool-assessment/v0.1
+
+Version: v0.1.0
+
+## Purpose
+
+**The tool assessment attestation references the results and metadata associated with tools used to assess software before or after its creation.** It's primary purpose is to provide an immutable attestation of tool assessment of a software so that it can be bundled with its provenance. This enables to mapping of build provenances to assessments on a per build basis and can reflect entire DevSecOps pipeline processes.
+
+ However, this attestation can be used to describe the assessment of any target using any tool for any purpose and is not restricted to pipelines.
+
+There are many existing predicates that describe the use of specific tool types. While these predicates are well defined, they are narrowly scoped. There should exist a predicate that is general enough to effectively attest the use of any tool that can be used if a tool type does not have a predicate type yet. The tool assessment attestation type aims to solve this.
+
+-   The [cyclonedx](cyclonedx.md) and [spdx](spdx.md) predicate types describe SBOM standards.
```

Comment:
I wonder if this is not too far from witness's commandrun attestor. @jkjell and others may have some thoughts there. If there's overlap, maybe we can consolidate the two.